lib/features/auth/presentation/bloc/user_config/user_config_cubit.dart

```diff
@@ -52,6 +56,12 @@ class UserConfigCubit extends Cubit<UserConfigState> {
       log.i("userId is null");
     }
   }
+  Future<void> setVoice(Map<String, dynamic> voice) async {
+    final prefs = await SharedPreferences.getInstance();
+    await prefs.setString(UserConfigConstants.prefKeyVoice, voice['name']);
+    emit(state.copyWith(selectedVoice: voice));
+  }
+
```

Comment:
Remove this and reuse the `setUserConfig` as mentioned in other comment

Reply:
copy

Reply:
can you help me in this because reusing the same is posing some errors like undefined name etc.

Reply:
Make sure to add it in `fromJson` and `toJson` for `UserConfigModel` class in `lib/features/auth/data/models/user_config_model.dart`